.github/workflows/omm-ci.yaml

```diff
@@ -59,8 +59,7 @@ jobs:
         image: postgres
         # Provide the env variables
         env:
-          POSTGRES_HOST: localhost
-          POSTGRES_DB: media_match
+          POSTGRES_DB: media_match_test
```

Comment:
Are we certain this DB is being used? If the docker image itself contains a database, and includes the instructions for creating the test database, and the docker run below doesn't have permission to talk to this one, do we need this copy?